teamforus/forus-backend
# Release v0.10.0

Changelog:
- Organization logo, change aspect ratio
- Unassigned vouchers, add pdf export option
- Notifications, dynamic from address for implementations
- Product store and update, 'expire_at' validation message updated
- testing and refactoring
- use 'validate_records' role instead of 'validator' role to check records validaty
- Add 'last_active_day' as model accessor
- Add implementation resource
